packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -84,6 +84,8 @@ export const enableDiscreteEventFlushingChange = true;
 // to the correct value.
 export const enableNewReconciler = __VARIANT__;
 
+export const enableRecursiveCommitTraversal = false;
```

Comment:
Should this be either true or in the dynamic values?

Reply:
See this comment https://github.com/facebook/react/pull/20094#issuecomment-717469662

Andrew's plan was to split iterative and recursive between old and new forks (once we merge the new fork into the old). For now, we're turning on iterative only for the new build (to see if that fixes the top line metrics).

Does make me a little uneasy that we don't have automated tests covering the new + recursive combination, but if we're able to merge new -> old soon, hopefully it's ok?

Reply:
Yeah ignore me, I'm a clown. This forces iterative mode which is what we want.